storage/storage_interface.ml

```diff
@@ -113,6 +113,15 @@ end
 type vdi = Vdi.t
 let vdi_pp : Format.formatter -> vdi -> unit = fun ppf vdi -> Format.fprintf ppf "%s" (Vdi.string_of vdi)
 
+module Vm : WRAPPEDSTRING = struct
+  type t = string [@@deriving rpcty]
```

Comment:
might be worth adding a comment here that this is actually a `domid`, at least for now.